### ISSUE TITLE
lgtv.py on

### ISSUE THREAD

**Issue by klattimer** [klattimer/LGWebOSRemote#6]
Turning on the TV via Wake on LAN is unreliable after a few minutes, this is probably because LG haven't gotten the state of the wifi hardware setup correctly when the TV is in standby and the setting is correctly enabled on the TV.

I'd appreciate someone confirming this bug for me if possible.

**Comment by akhan23wgu** [comment]
Is your TV running over Ethernet or WiFi?

**Comment by karlinnolabs** [comment]
@akhan23wgu yeah, when its on wired network it seems fine, but my TV is wifi only so thats where I noticed the issue - would you test again over wifi please?

**Comment by akhan23wgu** [comment]
I was running on an Access Point (Meraki MR33) for the past two weeks, and I did not notice any such disconnects. Attached a picture of the settings, pulled from my Meraki Dashboard, maybe that can help.

I am on a powerline adapter now, for the sake of having less WiFi congestion.

![image](https://user-images.githubusercontent.com/16216291/29079234-d913c362-7c19-11e7-8726-cf400d4afa0d.png)
